.github/workflows/ci.yml

```diff
@@ -138,12 +140,12 @@ jobs:
           Copy-Item -Path ".\tmp\tiny_tds-$gemVersion-$rubyArchitecture\ports" -Destination "." -Recurse
 
       - name: Setup MSSQL
-        uses: potatoqualitee/mssqlsuite@v1.7
+        uses: andyundso/setup-mssql@v1
```

Comment:
Worth bringing this into the rails-sqlserver github org?

Reply:
can do so in the next few days!